jshift-kit-vertx/src/main/java/io/jshift/vertx/generator/VertxGenerator.java

```diff
@@ -46,7 +46,7 @@
 public class VertxGenerator extends JavaExecGenerator {
 
   public VertxGenerator(GeneratorContext context) {
-    super(context, "vertx");
+    super(context, "io/jshift/vertx/generator");
```

Comment:
is this renaming required? If yes, can we avoid having `/` in our names

Reply:
What are we going to do with this renaming thing? It is affecting other PRs.

Reply:
I think that we should only change the `fabric8` tags. So if it is called `vertx` shouldn't be named `vertx` too? But for example, if it was `fabric8/vertx` then yes let's change it to `jshift/vertx`. Does it make sense?

Reply:
I think we should rename it only when it has references to fabric8, otherwise leave it as it is.